pkgs/development/python-modules/sqlalchemy/default.nix

```diff
@@ -25,7 +25,9 @@ buildPythonPackage rec {
 
   dontUseSetuptoolsCheck = true;
 
-  disabledTests = lib.optionals isPy35 [ "exception_persistent_flush_py3k "];
+  # disable mem-usage tests on mac, has trouble serializing pickle files
+  disabledTests = lib.optionals isPy35 [ "exception_persistent_flush_py3k "]
+    ++ lib.optionals stdenv.isDarwin [ "MemUsageWBackendTest" "MemUsageTest" ];
```

Comment:
I think these only fail on darwin Python3.8 https://hydra.nixos.org/job/nixpkgs/trunk/python37Packages.sqlalchemy.x86_64-darwin https://hydra.nixos.org/job/nixpkgs/trunk/python38Packages.sqlalchemy.x86_64-darwin. 
Upstream disabled `MemUsageTest` for windows, maybe we can ask the the same for darwin https://github.com/sqlalchemy/sqlalchemy/issues/4946

Reply:
most likely, but I don't have a darwin environment to repro this.